extensions/mssql/src/objectManagement/objectManagementService.ts

```diff
@@ -146,6 +125,16 @@ export class ObjectManagementService implements IObjectManagementService {
 			}
 		);
 	}
+	drop(connectionUri: string, objectUrn: string): Thenable<void> {
+		const params: contracts.DropObjectRequestParams = { connectionUri, objectUrn };
+		return this.client.sendRequest(contracts.DropObjectRequest.type, params).then(
+			r => { },
+			e => {
+				this.client.logFailedRequest(contracts.DropObjectRequest.type, e);
+				return Promise.reject(new Error(e.message));
```

Comment:
Why not reject the original error itself?

Reply:
done